spec/features/complexity_level_history_feature_spec.rb

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+feature 'Case history with complexity level' do
+  before do
+    test_strategy.switch!(:womens_estate, true)
+
+    allow(HmppsApi::ComplexityApi).to receive(:get_history).with(offender_no).and_return(history)
+    stub_signin_spo logged_in_user, [prison_code]
+    stub_offenders_for_prison prison_code, [offender]
+    stub_poms prison_code, [pom]
+
+    stub_request(:get, "#{ApiHelper::T3}/users/user").
+      to_return(body: { staffId: pom.staff_id, firstName: pom.first_name, lastName: pom.last_name }.to_json)
+
+    create(:allocation, prison: prison_code, nomis_offender_id: offender_no,
+           allocated_at_tier: case_info.tier,
+           created_by_name: created_by_name,
+           primary_pom_nomis_id: pom.staff_id, primary_pom_name: pom_name)
+
+    visit prison_allocation_history_path(prison_code, offender_no)
+  end
+
+  after do
+    test_strategy.switch!(:womens_estate, false)
+  end
+
+  let(:test_strategy) { Flipflop::FeatureSet.current.test! }
+  let(:now) { Time.zone.now }
+  let(:complexity_add_time) { now - 1.hour }
+  let(:complexity_change_time) { now + 1.hour }
+  let(:username) { 'user' }
+  let(:logged_in_user) { build(:pom) }
+  let(:pom) { build(:pom) }
+  let(:pom_name) { "#{pom.last_name}, #{pom.first_name}" }
+  let(:prison) { build(:womens_prison) }
+  let(:prison_code) { prison.code }
+  let(:case_info) { create(:case_information) }
+  let(:offender_no) { case_info.nomis_offender_id }
+  let(:offender) { build(:nomis_offender, offenderNo: offender_no) }
+  let(:created_by_name) { "#{logged_in_user.last_name}, #{logged_in_user.first_name}" }
+
+  context 'with 1 history record' do
+    let(:history) {
+      [{ level: 'high',
+         createdTimeStamp: complexity_add_time,
+         sourceUser: username }]
+    }
+
+    it 'has 1 prison section' do
+      expect(all('.govuk-grid-row').size).to eq(1)
+    end
+
+    it 'has a section with complexity and allocation' do
+      within '.govuk-grid-row:nth-of-type(1)' do
+        expect(page).to have_css('.govuk-heading-m', text: prison.name)
+        expect(all('.moj-timeline__item').size).to eq(2)
+      end
+    end
+
+    # have to put :js here as <CR> handled differently between Capybara and Rack::Test
```

Comment:
What is `<CR>`? I can't figure out what it means in this context – I don't recognise it as a HTML element.

Would be good to clarify this comment to make sure it more helpful/meaningful.